### DISCUSSION TITLE
[@types/ramda] 0.29.1 issues with propEq function

### DISCUSSION THREAD

**Discussion by cybertouchllc** [DefinitelyTyped/DefinitelyTyped#65347] [Issues with a @types package]
Looks like the arguments are flipped:

<img width="357" alt="Screenshot 2023-04-29 at 2 50 14 p m" src="https://user-images.githubusercontent.com/62496553/235323695-48e5fbf4-051b-450c-bf85-d319a18d89cc.png">

It should be:

```
export function propEq<string>(name: string,  val: any, obj: Record<string, any>): boolean
```

[Docs](https://ramdajs.com/docs/#propEq)

`ramda` version `0.29.0`
`@types/ramda` version `0.29.1`
`typescript` version `4.9.5`

[a comment by an automated account is not shown]

**Comment by Nicholaiii** [comment]
Are you misreading the docs? The types correspond to the example on the docs. Here's a REPL too: https://ramdajs.com/repl/?v=0.29.0#?propEq%28%27val%27%2C%20%27name%27%2C%20%7B%20foo%3A%202%20%7D%29%0A

**Comment by Harris-Miller** [comment]
@cybertouchllc the order of the `propEq` arguments were changed in 0.29.0
Release notes: https://github.com/ramda/ramda/releases/tag/v0.29.0

**Reply by cybertouchllc** [reply]
sure thing. Thanks. 
I was reading the docs on `devdocs` website which is outdated to ramda v0.27 :/
